taprpc/universerpc/universe.proto

```diff
@@ -179,6 +179,14 @@ service Universe {
     */
     rpc FetchSupplyCommit (FetchSupplyCommitRequest)
         returns (FetchSupplyCommitResponse);
+
+    /* tapcli: `universe fetchsupplyleaves`
```

Comment:
nit: do we need the `fetch` prefix in the command name? IMO just `supplyleaves` would probably be enough?